fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -3117,7 +3117,7 @@ public void testLoanScheduleWithInterestRecalculation_WITH_REST_SAME_AS_REPAYMEN
 
         Calendar todaysDate = Calendar.getInstance(Utils.getTimeZoneOfTenant());
         todaysDate.add(Calendar.DAY_OF_MONTH, -14);
-        final String LOAN_DISBURSEMENT_DATE = dateFormat.format(todaysDate.getTime());
+        final String loanDisbursementDateValue = dateFormat.format(todaysDate.getTime());
 
```

Comment:
This one has ...Value, others don't. Is there a difference that you had in mind?

Reply:
Updated